src/resolvers/ArticleResolver.ts

```diff
@@ -60,11 +60,27 @@ class ArticleResolver {
     return article['_doc'].shoutouts / (presentDate - article['_doc'].date.getTime()); // eslint-disable-line
   }
 
+  @FieldResolver((_returns) => Number)
+  async nsfw(@Root() article: Article): Promise<boolean> {
+    return ArticleRepo.checkProfanity(article['doc'].title); //eslint-disable-line
+  }
+
   @Mutation((_returns) => [Article])
   async refresh() {
     return ArticleRepo.refreshFeed();
   }
 
+  @Mutation((_returns) => Article)
```

Comment:
Potentially dangerous, if the same articleURL is provided on two separate mutations, this will crash.

Reply:
I think this is a limitation of using newman tests. They aren't good for checking internal functionality, and something like `jest` would definitely be much more normal to use.